release notes/v1.4.0.md

```diff
@@ -0,0 +1,211 @@
+k6 `v1.4.0` is here 🎉! This release includes:
+
+- Changes in the Browser module:
+  - `page.waitForRequest` for waiting on specific HTTP requests.
+  - `QueryAll` methods now return elements in DOM order.
+  - `locator.evaluate` and `locator.evaluateHandle` for executing JavaScript code in the page context with access to the matching element.
+  - `page.unroute(url)` and `page.unrouteAll` for removing routes registered with `page.route`.
+- Other notable changes:
+  - OpenTelemetry output graduated from experimental to stable status.
```

Comment:
Not a strong opinion, but should we list the last item first, rather than having a separate list?

```suggestion
- OpenTelemetry output graduated from experimental to stable status.
- Changes in the Browser module:
  - `page.waitForRequest` for waiting on specific HTTP requests.
  - `QueryAll` methods now return elements in DOM order.
  - `locator.evaluate` and `locator.evaluateHandle` for executing JavaScript code in the page context with access to the matching element.
  - `page.unroute(url)` and `page.unrouteAll` for removing routes registered with `page.route`.
```